repos/os/include/net/ipv4.h

```diff
@@ -132,7 +132,7 @@ class Net::Ipv4_packet
 		 *******************************/
 
 		Genode::size_t  version()     { return _version;                   }
-		Genode::size_t  header_length() { return _header_length / 4;       }
+		Genode::size_t  header_length() { return _header_length;           }
```

Comment:
Obviously, using division is wrong here (I wonder why it ever worked ;-)), but the history here is that the function shall return bytes and not 32bit-word count. So theoretically multiplication by four should do the right thing.